ydb/core/tx/replication/service/worker.cpp

```diff
@@ -196,6 +210,10 @@ class TWorker: public TActorBootstrapped<TWorker> {
         }
     }
 
+    void Handle(TEvWorker::TEvDataEnd::TPtr& ev) {
+        Forward(ev, Parent);
```

Comment:
Замени плз на:
```suggestion
        Send(ev->Forward(Parent));
```
чтобы в одном файле не было несколько способов сделать одно и то же.